internal/devserver/main/main.go

```diff
@@ -32,7 +34,26 @@ func main() {
 		os.Exit(1)
 	}
 
-	http.Handle(*servingPath, concatjs.ServeConcatenatedJS(*manifest, *base, nil /* realFileSystem */))
+	scripts := []string{RequireJs, "", ""}
```

Comment:
Anyone know of a better way to do this? Seems arrays are fixed length in Go so you can't push to them after declaration.